lib/product.rb

```diff
@@ -0,0 +1,20 @@
+class Product
+  attr_reader :category, :name, :unit_price, :quantity, :hoard
+  def initialize(category, name, unit_price, quantity)
+    @category = category
+    @name = name
+    @unit_price = unit_price
+    @quantity=  quantity.to_i
+    hoard = true
+  end
+
+  def total_price
+    @unit_price * @quantity
+  end
+
+  def is_hoarded?
```

Comment:
`is_hoarded?` should return the `@is_hoarded` instance variable

Reply:
Then we should have another method called `hoard` that changes that instance variable `@is_hoarded` to true
```
def hoard
  @is_hoarded = true
end
```